arrows/ocv/inpaint.h

```diff
@@ -0,0 +1,55 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#ifndef KWIVER_ARROWS_OCV_INPAINT_
+#define KWIVER_ARROWS_OCV_INPAINT_
+
+#include <arrows/ocv/kwiver_algo_ocv_export.h>
+
+#include <vital/algo/image_filter.h>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace ocv {
+
+/// OCV image inpainting process
+///
+/// Create a masked RGB image from a RGBA image
```

Comment:
This comment is not very clear.  You might want to write out more clearly what it means to inpaint and that the pixels from the alpha channels are used as a mask to indicate which pixel to paint.